<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>582</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="networkButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="arpButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anticrackButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fackapButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="devicesButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>641</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <zorder>label</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TitleBar">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
